cmd/src/users_verify.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"strings"
+)
+
+const SubCommand = "verify-emails"
+
+type VerifyEmailJson struct {
+	ID    string `json:"ID"`
+	Email string `json:"Email"`
+}
+
+func init() {
+	usage := `
+Examples:
+
+  Verify one or more user account(s):
+
+    	$ src users ` + SubCommand + ` -verify-json='[{"ID": "ID1", "Email": "email1"},...]'
+
+`
+
+	flagSet := flag.NewFlagSet(SubCommand, flag.ExitOnError)
+	usageFunc := func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage of 'src users %s':\n", flagSet.Name())
+		flagSet.PrintDefaults()
+		fmt.Println(usage)
+	}
+	var (
+		verifyJsonFlag = flagSet.String("verify-json", "",
+			`The JSON array of id-email descriptors of those user/emails that should be marked as verified. (required)`)
+		apiFlags = newAPIFlags(flagSet)
+	)
+
+	handler := func(args []string) error {
+		flagSet.Parse(args)
+
+		dec := json.NewDecoder(strings.NewReader(*verifyJsonFlag))
+
+		var verifyList []VerifyEmailJson
+		if err := dec.Decode(&verifyList); err != nil {
+			return err
+		}
+
+		vars := map[string]interface{}{}
+
+		// translate the provided index of `verifyList` into variables for its `id` and `email` payload
+		indexToKeys := func(idx int) (string, string) {
+			return fmt.Sprintf("id%09d", idx), fmt.Sprintf("email%09d", idx)
```

Comment:
This feels weird to me. I would probably prefer two separate maps here for readability but this comment might become obsolete if you use text/template as suggested in a different comment.